src/viewport.cpp

```diff
@@ -88,6 +88,9 @@
 #include "command_func.h"
 #include "network/network_func.h"
 #include "framerate_type.h"
+#include "depot_base.h"
+#include "tunnelbridge_map.h"
```

Comment:
includes seem unnecessary?

Reply:
It probably is; for the same reason as above.